Makefile

```diff
@@ -204,6 +204,8 @@ release: $(GITHUB_RELEASE)
 vendor: $(GO)
```

Comment:
running this locally shows lots of vendor files are deleted and were not "git added" to this PR please